integrations/access/servicenow/client.go

```diff
@@ -85,6 +85,18 @@ func NewClient(conf ClientConfig) (*Client, error) {
 		return nil, trace.Wrap(err)
 	}
 	client := resty.NewWithClient(defaults.Config().HTTPClient)
+	apiURL, err := url.Parse(conf.APIEndpoint)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	if apiURL.Scheme == "http" && !strings.HasPrefix(apiURL.Host, "127.0.0.1") {
```

Comment:
Can you do an exact match against `apiURL.Hostname()` instead? And compare against `127.0.0.1` or `localhost`.

This currently returns true for hosts like `127.0.0.1.evil.com`

Reply:
Something like this?